core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -85,7 +88,7 @@ private[ui] class ExecutorsPage(
             </span>
           </th>
           {if (logsExist) <th class="sorttable_nosort">Logs</th> else Seq.empty}
-          {if (threadDumpEnabled) <th class="sorttable_nosort">Thread Dump</th> else Seq.empty}
```

Comment:
I not remove that actually, Because for the lost executor, it can't do dump, 
So I change that column head to "Status", for executor live, will show dump, for executor lost, will show a "Removed" status.